ocaml/utils/clflags.mli

```diff
@@ -199,6 +199,8 @@ val afl_instrument : bool ref
 val afl_inst_ratio : int ref
 val function_sections : bool ref
 val probes : bool ref
+val use_cached_startup : string option ref
```

Comment:
These flags can be in `driver/flambda_backend_flags.ml` because they are only used in the backend.